add_on_usage.go

```diff
@@ -0,0 +1,126 @@
+package recurly
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+)
+
+// AddOnUsageService manages the interactions for add-on usages.
+type AddOnUsageService interface {
+	// List returns a pager to paginate usages for add-on in subscription. PagerOptions are used to
+	// optionally filter the results.
+	//
+	// https://dev.recurly.com/docs/list-add-ons-usage
+	List(subUUID, addOnCode string, opts *PagerOptions) Pager
+
+	// Create creates a new usage for add-on in subscription.
+	//
+	// https://dev.recurly.com/docs/log-usage
+	Create(ctx context.Context, subUUID, addOnCode string, usage AddOnUsage) (*AddOnUsage, error)
+
+	// Get retrieves an usage. If the usage does not exist,
+	// a nil usage and nil error are returned.
+	//
+	// https://dev.recurly.com/docs/lookup-usage-record
+	Get(ctx context.Context, subUUID, addOnCode, usageId string) (*AddOnUsage, error)
```

Comment:
`usageId` should be named `usageID` (multiple instances in PR)

Reply:
Got it, thanks for info - changed